firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -174,6 +177,7 @@ FetchResponse fetch(
       Map<String, String> analyticsUserProperties,
       String lastFetchETag,
       Map<String, String> customHeaders,
+      Instant firstOpenTime,
```

Comment:
Thinking aloud: it doesn't make sense to merge this in with the other `analyticsUserProperties` since we want to send it as its own request param.

Reply:
Yes, and the reason we want to send it in its own request param is because analyticsUserProperties in fetch traditionally contains custom user-defined properties only. And we want to keep, and highlight, that distinction. It also lets us explicitly call out the information that we are sending up to the server.